e2e/cypress/integration/api/iam/01_projects_api.spec.ts

```diff
@@ -319,3 +333,38 @@ describe('projects API', () => {
     })
   }
 })
+
+// TODO fix
+// function waitForSuccessfulApply() {
+//   cy
+//     .request({
+//       headers: {
+//         'api-token': admin_token      
+//       },
+//       method: 'GET',
+//       url: '/apis/iam/v2beta/apply-rules'
+//     })
+//     .then((resp: Cypress.ObjectLike) => {
+//       if (resp.body.percentage_complete == 1 && resp.body.state == "not_running" && !resp.body.failed)
```

Comment:
should the percentage complete be `100`? or is that an oddly named variable?

Reply:
Yeah this might require some work on the ingest-service side. Right now percentage_complete returns as `0` or `1` (at least when dealing with very small data sets like in this test)